tests/unittests/OperatorTest.cpp

```diff
@@ -64,6 +64,270 @@ static Tensor createTensorConditionallyQuantized(ElemKind T,
   return isQuantizedElemKind(T) ? Tensor(T, dims, 1.0, 0) : Tensor(T, dims);
 }
 
+// Helper to test SpaceToDepth using \p DTy.
+template <typename DataType>
+static void testSpaceToDepthBlock3(glow::PlaceholderBindings &bindings,
+                                   glow::Module &mod, glow::Function *F,
+                                   glow::ExecutionEngine &EE, ElemKind DTy) {
+  unsigned blockSize = 3;
+  auto *in = createPlaceholderConditionallyQuantized(mod, DTy, {1, 2, 6, 6},
+                                                     "in", false);
+  auto *tri = F->createTranspose("sptdTransposeIn", in, {0, 2, 3, 1});
+  auto *stdn = F->createSpaceToDepth("spacetodepth", tri, blockSize);
+  auto *tro = F->createTranspose("sptdTransposeOut", stdn, {0, 3, 1, 2});
+  auto *save = F->createSave("save", tro);
+  auto *result = bindings.allocate(save->getPlaceholder());
+
+  /*
+    Example for first batch.
+  FROM:
+  C0:             C1:
+  [0  1  2  3  16 17]    [ 0  -1  -2  -3  -16 -17]
+  [4  5  6  7  18 19]    [-4  -5  -6  -7  -18 -19]
+  [8  9  10 11 20 21]    [-8  -9  -10 -11 -20 -21]
+  [12 13 14 15 22 23]    [-12 -13 -14 -15 -22 -23]
+  [24 25 26 27 28 29]    [-24 -25 -26 -27 -28 -29]
+  [30 31 32 33 34 35]    [-30 -31 -32 -33 -34 -35]
+
+  TO:
+  C = 0
+[0,3]
```

Comment:
Please add 2 spaces to lines from 94 to 163, so that all the lines in the comment are aligned the same way.